### PR TITLE
DM-6999: Logging framework migration 

### PR DESCRIPTION
**Pull request by hchiang2**

### CODE REVIEW

File: python/lsst/meas/deblender/deblend.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ def addSchemaKeys(self, schema):
             'deblend_hasStrayFlux', type='Flag',
             doc=('This source was assigned some stray flux'))
 
-        self.log.logdebug('Added keys to schema: %s' % ", ".join(str(x) for x in (
+        self.log.trace('Added keys to schema: %s', ", ".join(str(x) for x in (
                     self.nChildKey, self.psfKey, self.psfCenterKey, self.psfFluxKey,
```
Comment (r-owen) [inline python/lsst/meas/deblender/deblend.py]: Why the change in level from debug to trace?

File: python/lsst/meas/deblender/baseline.py [hunk 2]
```diff
@@ -300,7 +300,7 @@ def deblend(footprint, maskedImage, psf, psffwhm,
         # FIXME -- just within the bbox?
         stats = afwMath.makeStatistics(varimg, mask, afwMath.MEDIAN)
         sigma1 = math.sqrt(stats.getValue(afwMath.MEDIAN))
-        log.logdebug('Estimated sigma1 = %f' % sigma1)
+        log.trace('Estimated sigma1 = %f', sigma1)
```
Comment (r-owen) [inline python/lsst/meas/deblender/baseline.py]: Why the change in level from debug to trace (here and above on line 276: `Log.TRACE`)

File: python/lsst/meas/deblender/deblend.py (hunk 1 above)
Comment (hchiang2) [inline python/lsst/meas/deblender/deblend.py]: It was discussed previously [here](https://github.com/lsst/meas_deblender/commit/6428a77262cdc9a9fe39e17da8a6bca9735b1c77) (and noted in the commit message).   For this logger named "meas_deblender.baseline", two tracing levels are currently used in master: a less verbose `log.log(-9, ...)` and a more verbose `log.logdebug(...)`. So the former maps to `log.debug()` and the latter maps to `log.trace()` in the conversion.
